src/main/java/org/broadinstitute/hellbender/engine/filters/AlignmentAgreesWithHeaderReadFilter.java

```diff
@@ -9,7 +9,11 @@
 import java.io.Serializable;
 
 /**
- * Checks to ensure that the alignment of each read makes sense based on the contents of the header.
+ * Filter out reads where the alignment does not make sense based on the contents of the header:
```

Comment:
Replaced. This compares to the sequence dictionary which is in the header for the source of reads.